WaterBot/Framework/WaterBotControler.cs

```diff
@@ -155,7 +153,6 @@ public void water(Point tile)
                 Game1.player.EndEmoteAnimation();
             }
 
-            Game1.player.FarmerSprite.setOwner(Game1.player);
```

Comment:
There simply doesn't seem to be an equivalent method for this in 1.6. I have not noticed any adverse effects after removing it.

Reply:
Somehow I missed it, but `SetOwner` exists... fixed in latest commit.